lib/foreman_ansible_core/runner/ansible_runner.rb

```diff
@@ -104,7 +106,9 @@ def write_playbook
       end
 
       def start_ansible_runner
-        command = ['ansible-runner', 'run', @root, '-p', 'playbook.yml']
+        env = {}
+        env['FOREMAN_DISABLE'] = '1' if @rex_command
+        command = [env, 'ansible-runner', 'run', @root, '-p', 'playbook.yml']
```

Comment:
Is this a correct way to set env variables? passing the env hash seems to be quite wild.

Reply:
Not sure if it is the correct way, but it is the way we did it for ansible-playbook

Reply:
Ignore me, I was using wrong callback :) It works... still looks weird, but works xD